### ISSUE TITLE
Write some python to convert the sample svg file into instructions for our existing code base.

### ISSUE THREAD

**Issue by JackBuck** [JackBuck/Robo-Plot#32]
This does not necessarily have to be done by the *'Research SVG Files'* milestone deadline.
- The code base is (at the time of writing) not ready
- We currently have no official example svg containing all the elements we must recognise

**Comment by JackBuck** [comment]
This issue has really become one with #20, since the svgpathtools library does the work which separates them!
Both issues are waiting on finding a way to get around the slowness of calling the Path.ilength() method.

**Comment by JackBuck** [comment]
Closing since superseded by issue #20
